### ISSUE TITLE
[BUG] Valid YAML is skipped because of YAML error

### ISSUE THREAD

**Issue by KucharczykL** [Mara-Li/mkdocs_obsidian_publish#5]
**Describe the bug**
A file with a valid YAML is skipped because it is reported to have a YAML error.

**To Reproduce**
1. Create a new file with this content:
```yaml
---
name: "Test: Test"
---
```
2. Convert with `obs2mk`.
3. See error.

**Log** 
```bash
> obs2mk --git --vault
[16:32:27] STARTING CONVERT [ALL] OPTIONS :
- NO PUSH
- SHARE ENTIRE VAULT [IGNORE SHARE STATE]
Skip $FILE because of YAML error.
```

**Additional context**
obs2mk version 1.2.5

**Comment by Mara-Li** [comment]
Strange. I need to check. As I said, I don't have any control on pyYAML.

**Comment by Mara-Li** [comment]
It's strange : I don't have any error on my file.

**Comment by Mara-Li** [comment]
Okay, I pretty sure I found the error : in python, the multiple `"` inside the yaml, are removed. So, the script read something as : `name: test: test`

**Comment by Mara-Li** [comment]
Normally, the 1.2.6 version repear this problem

**Comment by KucharczykL** [comment]
It appears to be fixed in 1.2.6! Thank you again.
